app/views/shared/_claim.html.haml

```diff
@@ -1,9 +1,24 @@
 %section
+
+  .certification-info-row
+    .bold-xsmall{ style: 'margin-bottom: 5px;' }
+      = t('.certification_box_title')
+    %div.bold-xsmall
```

Comment:
cheers dude, this had just been left in from me mucking about using ```%div{style: 'whatever:init;'}``` previously. incidentally i need to talk to you about some styling on this and the main form PPE and NPW sections. til monday